ui/raidboss/data/07-dt/raid/r7n.txt

```diff
@@ -1,9 +1,274 @@
+157.1 "Brutish Swing" Ability { id: "A535", source: "Brute Abombinator" }
+158.1 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+163.1 "Stoneringer: The Stabbing" Ability { id: "A533", source: "Brute Abombinator" }
+173.0 "Brutish Swing" Ability { id: "A539", source: "Brute Abombinator" }
+175.9 "Brutish Swing" Ability { id: "A541", source: "Brute Abombinator" }
+177.1 "Brutish Swing" Ability { id: "A53F", source: "Brute Abombinator" }
+181.7 "Glower Power" Ability { id: "A546", source: "Brute Abombinator" }
+182.9 "Glower Power" Ability { id: "A94B", source: "Brute Abombinator" }
+183.0 "Electrogenetic Force" Ability { id: "A547", source: "Brute Abombinator" }
+190.0 "Abominable Blink" Ability { id: "A542", source: "Brute Abombinator" }
+191.0 "Abominable Blink" Ability { id: "A892", source: "Brute Abombinator" }
+192.0 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+199.0 "Revenge of the Vines" Ability { id: "A543", source: "Brute Abombinator" }
+201.0 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+206.0 "Stoneringer: The Stabbing" Ability { id: "A533", source: "Brute Abombinator" }
+216.9 "Sporesplosion" Ability { id: "A544", source: "Brute Abombinator" }
+223.0 "Sporesplosion" #Ability { id: "A545", source: "Brute Abombinator" }
+225.0 "Sporesplosion" #Ability { id: "A545", source: "Brute Abombinator" }
+225.4 "Brutish Swing" Ability { id: "A53D", source: "Brute Abombinator" }
+227.0 "Sporesplosion" #Ability { id: "A545", source: "Brute Abombinator" }
+228.3 "Brutish Swing" Ability { id: "A541", source: "Brute Abombinator" }
+229.5 "Brutish Swing" Ability { id: "A53F", source: "Brute Abombinator" }
+234.1 "Glower Power" Ability { id: "A546", source: "Brute Abombinator" }
+235.3 "Glower Power" Ability { id: "A94B", source: "Brute Abombinator" }
+235.4 "Electrogenetic Force" Ability { id: "A547", source: "Brute Abombinator" }
+239.1 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+242.1 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+248.2 "Powerslam" Ability { id: "A548", source: "Brute Abombinator" }
+
+# Third arena
+269.8 "Stoneringer 2: Stoneringers" Ability { id: "A54A", source: "Brute Abombinator" }
+278.7 "Spore Sac" Ability { id: "A529", source: "Brute Abombinator" }
+280.8 "--sync--" Ability { id: "A516", source: "Brute Abombinator" }
+283.8 "Spore Sac" Ability { id: "A52A", source: "Brute Abombinator" }
+286.4 "Brutish Swing" Ability { id: "A54C", source: "Brute Abombinator" }
+289.2 "Brutish Swing" Ability { id: "A550", source: "Brute Abombinator" }
+290.2 "Brutish Swing" Ability { id: "A54F", source: "Brute Abombinator" }
+291.3 "Revenge of the Vines" Ability { id: "A638", source: "Brute Abombinator" }
+295.1 "Pollen" Ability { id: "A52B", source: "Brute Abombinator" }
+298.3 "Lashing Lariat" Ability { id: "A553", source: "Brute Abombinator" }
+298.7 "Lashing Lariat" Ability { id: "A554", source: "Brute Abombinator" }
```

Comment:
Here and elsewhere, are the Lariats fixed or random?

Reply:
```suggestion
298.3 "Lashing Lariat" Ability { id: ["A551", "A553"], source: "Brute Abombinator" }
298.7 "Lashing Lariat" Ability { id: ["A552", "A554"], source: "Brute Abombinator" }
```
Never mind, I checked my logs from week 2 and this is indeed random from this point on. Which Lariat is used is dependent on which hand the weapon for the prior Brutish Swing is in, so those will need adjustment as well.